stacks/wordpress/README.md

```diff
@@ -13,8 +13,8 @@ This DigitalOcean Marketplace Kubernetes 1-Click installs [WordPress](https://gi
 
```

Comment:
For the wordpress stack, I don't see any Helm chart version change in the [deploy.sh](https://github.com/v-bpastiu/marketplace-kubernetes/blob/v-bpastiu/upgrade-starterkit-1-click-apps/stacks/wordpress/deploy.sh#L16) script. It's still using the old version of **13.0.22**.

Reply:
Latest commit has everything